gno/p/dao_interfaces/modules.gno

```diff
@@ -44,7 +44,7 @@ type IRolesModule interface {
 	Render(path string) string
 	GetMemberRoles(address std.Address) []string
 	HasRole(address std.Address, role string) bool
-	NewRole(roleName string)
+	NewRole(roleName string, permissionsJSON string)
```

Comment:
we shouldn't take json in this func, if we need to pass json through the interface, create a `NewRoleJSON` func instead

JSON un/marshaling should be only at the edge